<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageProcessingWindow</class>
 <widget class="QMainWindow" name="ImageProcessingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Image Processing</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="photo">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>100</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>img/kontrolny3.tiff</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>620</y>
      <width>800</width>
      <height>20</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>640</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Choose File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>680</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save File</string>
    </property>
   </widget>
   <widget class="QLabel" name="pathLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>File Path</string>
    </property>
   </widget>
   <widget class="QComboBox" name="sizeBox">
    <property name="geometry">
     <rect>
      <x>145</x>
      <y>682</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4x</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8x</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="resizeLabel">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>650</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select size:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixelLabel">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>650</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected pixel:</string>
    </property>
   </widget>
   <widget class="QLabel" name="pixelPhoto">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>670</y>
      <width>81</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>PIXEL IMG</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="textBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>682</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="pixelValueLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>650</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Pixel Value:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="valueButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>720</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Value</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
